chainer/utils/experimental.py

```diff
@@ -0,0 +1,103 @@
+import chainer
+import warnings
+
+
+def experimental(api_name):
+    """A function for marking APIs as experimental.
```

Comment:
Start a docstring of function from a verb (of the third-person singular form), like this: `Declares that user is using an experimental feature.`